elasticsearch.md

```diff
@@ -920,6 +920,46 @@ After adding the `filter` rule \(using the block duplication strategy\).
       kibana_index: ".kibana_@{user}"
 ```
 
+#### `response_fields`
+
+This rule allows filtering Elasticsearch responses using defined list of fields. It works in very similar way to fields rule. However it **doesn't make use of Field Level Security \(FLS\)** and could be applied to every response returned by Elasticsearch.
+
+It could be configured in two modes: *whitelist* passing only defined fields or *blacklist* filtering out defined fields.
+
+**Blacklist mode**
+
+Specifies which fields should be filtered out with `~` (other fields from mapping become allowed implicitly). Example:
+
+`fields: ["~excluded_fields_prefix_*", "~excluded_field", "~another_excluded_field.nested_field"]`
+
+Return documents but deprived of the fields that:
```

Comment:
let's don't talk about document in this rule. This rule filters response fields, not document fields (like `fields` rule)